flake.nix

```
@@ -3,12 +3,13 @@
   inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
   inputs.nixpkgs-unstable.url = "github:nixos/nixpkgs/nixpkgs-unstable";
   inputs.prybar.url = "github:replit/prybar";
+  inputs.java-language-server.url = "github:replit/java-language-server";
```

```suggestion
  inputs.java-language-server.url = "github:replit/java-language-server";
  inputs.java-language-server.inputs.nixpkgs.follows = "nixpkgs";
```
Ideally, we use the same nixpkgs as everything else  or "nixpkgs-unstable" to limit the number of nixpkgs checkouts we have to evaluate.